extensions/sql-database-projects/src/dialogs/publishDatabaseQuickpick.ts

```diff
@@ -83,8 +120,7 @@ export async function launchPublishDatabaseQuickpick(project: Project): Promise<
 
 
 	// 4. Modify sqlcmd vars
-	// TODO@chgagnon: Concat ones from publish profile
-	let sqlCmdVariables = Object.assign({}, project.sqlCmdVariables);
+	let sqlCmdVariables = Object.assign({}, project.sqlCmdVariables, publishProfile?.sqlCmdVariables);
```

Comment:
add comment that if the same sqlcmd variable is specified in both the sqlproj and the profile, the values from the profile will be used